modules/dtj/Animal/lib/Horse.pm

```diff
@@ -0,0 +1,145 @@
+package Horse;
+
+use 5.006;
+use strict;
+use warnings;
+
+use parent qw(Animal);
+
+sub sound { 'neigh' }
+
+=head1 NAME
+
+Horse - The great new Horse!
+
+=head1 VERSION
+
+Version 0.01
+
+=cut
+
+our $VERSION = '0.01';
+
+
+=head1 SYNOPSIS
+
+Quick summary of what the module does.
+
+Perhaps a little code snippet.
+
+    use Horse;
+
+    my $foo = Horse->new();
+    ...
+
+=head1 EXPORT
+
+A list of functions that can be exported.  You can delete this section
+if you don't export anything, such as for a purely object-oriented module.
+
+=head1 SUBROUTINES/METHODS
+
+=head2 function1
+
+=cut
+
+sub function1 {
```

Comment:
Yeah I spotted them but I also noticed that they were in some modules from earlier on as well and I was questioning if I should do them in this pull request or leave it and do it separately so that the diffs weren't polluted with deletions. However I guess right now they're polluted with pointless stuff so I didn't really save on much.